mavros/src/plugins/imu.cpp

```diff
@@ -0,0 +1,501 @@
+/**
+ * @brief IMU and attitude data parser plugin
+ * @file imu.cpp
+ * @author Vladimir Ermakov <vooon341@gmail.com>
+ *
+ * @addtogroup plugin
+ * @{
+ */
+/*
+ * Copyright 2013-2017 Vladimir Ermakov.
+ *
+ * This file is part of the mavros package and subject to the license terms
+ * in the top-level LICENSE file of the mavros repository.
+ * https://github.com/mavlink/mavros/tree/master/LICENSE.md
+ */
+
+#include <cmath>
+#include <mavros/mavros_plugin.h>
+#include <eigen_conversions/eigen_msg.h>
+
+#include <sensor_msgs/Imu.h>
+#include <sensor_msgs/MagneticField.h>
+#include <sensor_msgs/Temperature.h>
+#include <sensor_msgs/FluidPressure.h>
+#include <geometry_msgs/Vector3.h>
+
+namespace mavros {
+namespace std_plugins {
+/**
+ * Gauss to Tesla coeff
+ */
+static constexpr double GAUSS_TO_TESLA = 1.0e-4;
+/**
+ * millTesla to Tesla coeff
+ */
+static constexpr double MILLIT_TO_TESLA = 1000.0;
+/**
+ * millRad/Sec to Rad/Sec coeff
+ */
+static constexpr double MILLIRS_TO_RADSEC = 1.0e-3;
+/**
+ * millG to m/s**2 coeff
+ */
+static constexpr double MILLIG_TO_MS2 = 9.80665 / 1000.0;
+/**
+ * millBar to Pascal coeff
+ */
+static constexpr double MILLIBAR_TO_PASCAL = 1.0e2;
+/**
+ * Radians to degrees
+ */
```

Comment:
Why? Why that large format for single line comment? `//!` is also interpreted by doxygen!

Reply:
Yeah you are right, I just wanted to keep it consistent. But makes more sense the shorter format.